### DISCUSSION TITLE
New configuration string (optional)

### DISCUSSION THREAD

**Discussion by saintchoise** [TownyAdvanced/Towny#5364] [Ideas]
Add to config.yml optional feature: toggle_pvp_warmup_time. Disabled # by default.

After specifying the time in seconds, after writing the command "/t toggle pvp on", the PvP will be turned on only after the specified time.
Ideally, in the chat, all outsiders in the town will write "Turn on PVP in %town in %x seconds!" (the text can be changed in the language config).

Thanks in advance for your reply.

**Comment by LlmDl** [comment]
We don't do feature requests in the Discussions. We already have a pvp toggle cooldown/warmup. Are you asking for something different to what we already have?

**Reply by saintchoise** [reply]
Sorry for using the wrong section.

However, I'm talking about a function that would show in the chat to outsiders in the town that the townspeople have turned on pvp. The stranger will have a few seconds (configurable in the config) to realize that they will be fighting with him now.
And since this function is useful for most servers with bias towards PVE audience, so it will be disabled by default and configured at the request of the administrator in the config.
The current setting changes exactly the cooldown on the command itself, and not on the fact of changing pvp.
